extensions/elasticsearch/ActiveQuery.php

```diff
@@ -215,7 +215,8 @@ public function one($db = null)
             /* @var $class ActiveRecord */
             $class = $this->modelClass;
             $model = $class::instantiate($result);
-            $class::populateRecord($model, $result);
+            $_class = get_class($model);
```

Comment:
Variable name needs to be changed. We never use leading underscore in local variable names.